### DISCUSSION TITLE
Vite environment variables not accessible with vercel dev?

### DISCUSSION THREAD

**Discussion by codeallthethingz** [vercel/vercel#9156] [Help]
When I run my project trying to access `process.env.VITE_VERCEL_URL` I get a runtime error

`Uncaught ReferenceError: process is not defined`

It populates `process.env.NODE_ENV` just fine.  

This is a Vue/Vite project.  I created a minimal reproducible build here:

https://github.com/codeallthethingz/reproduction-missing-env

Vercel version - Vercel CLI 28.10.1

**Comment by amyegan** [comment]
Hi @codeallthethingz! Does it work if you change `process.env` to `import.meta.env` instead? I think that is the expected way to [access env variables in Vite](https://vitejs.dev/guide/env-and-mode.html).
